src/main/migrations/changesets/db_schema.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+    <changeSet author="scottfrazer" id="db_schema_other_table_alldb">
+        <createTable tableName="WORKFLOW_EXECUTION">
+            <column autoIncrement="true" name="WORKFLOW_EXECUTION_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_WORKFLOW_EXECUTION"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_UUID" type="VARCHAR(100)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="WDL_URI" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="STATUS" type="VARCHAR(50)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="START_DT" type="datetime">
+                <constraints nullable="false"/>
+            </column>
+            <column name="END_DT" type="datetime"/>
+        </createTable>
+
+        <createTable tableName="EXECUTION">
+            <column autoIncrement="true" name="EXECUTION_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_EXECUTION"/>
+            </column>
+            <column name="WORKFLOW_EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="CALL_FQN" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="STATUS" type="VARCHAR(255)">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+
+        <createTable tableName="JES_JOB">
+            <column autoIncrement="true" name="JES_JOB_ID" type="INT">
+                <constraints primaryKey="true" primaryKeyName="PK_JES_JOB"/>
+            </column>
+            <column name="EXECUTION_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="JES_ID" type="INT">
+                <constraints nullable="false"/>
+            </column>
```

Comment:
Sorry if this is a dumb question as I haven't poked at the JES proto wrappers, but are JES ids really numeric and not UUIDs?